/joint_state_publisher:
  ros__parameters:
    joint_names: ["shoulder_pan_joint", "shoulder_lift_joint", "elbow_joint", "wrist_1_joint", "wrist_2_joint", "wrist_3_joint"]
    joint_positions: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    publish_rate: 10